<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileArchiveOptionsClass</class>
 <widget class="QWidget" name="FileArchiveOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>43</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chbLocation">
     <property name="text">
      <string>Change the location of the file archive (restart required)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdtLocation" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblLocation">
        <property name="text">
         <string>Location:</string>
        </property>
        <property name="buddy">
         <cstring>lneLocation</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneLocation">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tlbLocation">
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>chbLocation</tabstop>
  <tabstop>lneLocation</tabstop>
  <tabstop>tlbLocation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chbLocation</sender>
   <signal>toggled(bool)</signal>
   <receiver>wdtLocation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
